Tests/Test/WebTestCaseTest.php

```diff
@@ -324,9 +323,6 @@ public function testLoadFixturesFilesPaths()
         );
     }
 
-    /**
-     * @QueryCount(100)
```

Comment:
Here is an example with this branch: https://github.com/alexislefebvre/LiipFunctionalTestBundle/tree/fix-bug-with-array-parameters-annotation-error

Tests fail: https://travis-ci.org/alexislefebvre/LiipFunctionalTestBundle/jobs/104322732

Maybe there is a better way to read annotations from a Bundle? http://stackoverflow.com/questions/29089757/can-i-implement-my-own-symfony2-annotations-easily/29095296#29095296